tests/Rector/Contrib/PHPUnit/SpecificMethodInternalTypeRector/Correct/correct.php.inc

```diff
@@ -4,35 +4,35 @@ final class MyTest extends \PHPUnit\Framework\TestCase
-        $this->assertInternalType('numeric', $something);
-        $this->assertNotInternalType('numeric', $something);
-        $this->assertInternalType('integer', $something);
-        $this->assertNotInternalType('integer', $something);
-        $this->assertInternalType('float', $something);
-        $this->assertNotInternalType('float', $something);
-        $this->assertInternalType('bool', $something);
-        $this->assertNotInternalType('bool', $something);
-        $this->assertInternalType('object', $something);
-        $this->assertNotInternalType('object', $something);
-        $this->assertInternalType('resource', $something);
-        $this->assertNotInternalType('resource', $something);
-        $this->assertInternalType('scalar', $something);
-        $this->assertNotInternalType('scalar', $something);
-        $this->assertInternalType('callable', $something);
-        $this->assertNotInternalType('callable', $something);
-        $this->assertTrue(is_file($something));
-        $this->assertFalse(is_file($something));
-        $this->assertTrue(is_dir($something));
-        $this->assertFalse(is_dir($something));
-        $this->assertTrue(is_null($something));
-        $this->assertFalse(is_null($something));
-        $this->assertTrue(is_a($something));
-        $this->assertFalse(is_a($something));
-        $this->assertTrue(is_link($something));
-        $this->assertFalse(is_link($something));
+        $this->assertInternalType('string', $something, 'third argument');
+        $this->assertNotInternalType('string', $something, 'third argument');
+        $this->assertInternalType('array', $something, 'third argument');
+        $this->assertNotInternalType('array', $something, 'third argument');
+        $this->assertInternalType('numeric', $something, 'third argument');
+        $this->assertNotInternalType('numeric', $something, 'third argument');
+        $this->assertInternalType('integer', $something, 'third argument');
+        $this->assertNotInternalType('integer', $something, 'third argument');
+        $this->assertInternalType('float', $something, 'third argument');
+        $this->assertNotInternalType('float', $something, 'third argument');
+        $this->assertInternalType('bool', $something, 'third argument');
+        $this->assertNotInternalType('bool', $something, 'third argument');
+        $this->assertInternalType('object', $something, 'third argument');
+        $this->assertNotInternalType('object', $something, 'third argument');
+        $this->assertInternalType('resource', $something, 'third argument');
+        $this->assertNotInternalType('resource', $something, 'third argument');
+        $this->assertInternalType('scalar', $something, 'third argument');
+        $this->assertNotInternalType('scalar', $something, 'third argument');
+        $this->assertInternalType('callable', $something, 'third argument');
+        $this->assertNotInternalType('callable', $something, 'third argument');
```

Comment:
As for tests, maybe it's not neccessary to test every single keywords with all the avilable options.

What is more important are all [relevant cases, aka npaths or cyclomatic complexity](https://www.tomasvotruba.cz/blog/2017/08/28/shopsys-spriker-and-sylius-under-static-analysis/#3-cyclomatic-complexity).

Less test lines would make it easier to read, review and debug later.

In this case, something like this would be enough:

```php
$this->assertInternalType('int', $something);
$this->assertInternalType('integer', $something);

// with extra arguments
$this->assertInternalType('callable', $something, 'third argument');
$this->assertNotInternalType('callable', $something, 'third argument');

// should keep untouched
$this->assertTrue(is_file($something));
```


What do you think?

Reply:
100% agreed. Gonna reduce the verbosity of our tests. Btw, may you help me with failed tests? I could not figure out what is wrong.

Reply:
Sure, let's look on it.

To prevent waiting to Travis (it 8 minutes now, I'll try to decrease it!), just run PHPStan locally:

```bash
vendor/bin/phpstan.phar analyse packages src tests --level max --configuration phpstan.neon
```

**or better simulate all Travis tasks - tests, coding standards and static analysis**

```
composer all
```

The error is probably related to https://github.com/rectorphp/rector/pull/199/files#diff-94e7c72d7cd582d2de7ef3b1b4eda92aR100
Where `$methodCallNode` is already types in method argument: 

```php
/** @var FuncCall $methodCallNode */
$isFunctionNode = $methodCallNode->args[0]->value;
```

This should fix that:

```php
/** @var FuncCall $isFunctionNode */
$isFunctionNode = $methodCallNode->args[0]->value;
```